gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/client.py.j2

```diff
@@ -476,6 +485,22 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         )
         {% endif %}
 
+{#
+    Add API Version to metadata as per https://github.com/aip-dev/google.aip.dev/pull/1331.
+    When using this macro, ensure the calling template
+    generates the following lines:
+        try:
+            from google.api_core import version_header
+            HAS_GOOGLE_API_CORE_VERSION_HEADER = True  # pragma: NO COVER
+        except ImportError:  # pragma: NO COVER
+            HAS_GOOGLE_API_CORE_VERSION_HEADER = False
+#}
+    {% if service.version %}
```

Comment:
I think we should use the same macro as we use for the standard templates, instead of duplicating code. Is there any way we can import that file here?

(I know we've had similar discussions before in previous PRs and we wound up going with duplication for the sake of expediency, but it makes the code more brittle.)

Reply:
See https://github.com/googleapis/gapic-generator-python/pull/1999#discussion_r1587913464

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/109edd462b49315450ca2d6672dc408069834bdc

I created `_shared_macros.j2` and used a symlink to include the macros in ads templates

Reply:
This does not show up as a symlink; see comment below.